src/controllers/controlpickermenu.cpp

```diff
@@ -737,6 +737,9 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
     addControl("[AutoDJ]", "skip_next",
                tr("Auto DJ Skip Next"),
                tr("Skip the next track in the Auto DJ queue"), autodjMenu);
+    addControl("[AutoDJ]", "add_random_track",
+               tr("Auto DJ Add Random Track"),
+               tr("Add random track to the Auto DJ queue"), autodjMenu);
```

Comment:
I am sure this needs to be carefully revised.

Reply:
```suggestion
               tr("Add a random track to the Auto DJ queue"), autodjMenu);
```